.github/CONTRIBUTING.md

```diff
@@ -7,10 +7,12 @@ We're so glad you asked!
 * [Reporting Bugs](#reporting-bugs)
 * [Improving Documentation](#improving-documentation)
 * [Contributing Code](#contributing-code)
-   * [Improving existing katas](#improving-existing-katas)
+   * [The structure of a kata/tutorial](#the-structure-of-a-katatutorial)
+   * [Adding new tasks to existing katas](#adding-new-tasks-to-existing-katas)
+   * [Contributing workbooks](#contributing-workbooks)
    * [Contributing new katas](#contributing-new-katas)
    * [Style guide](#style-guide)
-   * [Updating the Katas to the new QDK version](#updating-the-Katas-to-the-new-QDK-version)
+   * [Updating the Katas to the new QDK version](#updating-the-katas-to-the-new-qdk-version)
```

Comment:
```suggestion
   * [Updating the katas to the new QDK version](#updating-the-katas-to-the-new-qdk-version)
```

Reply:
This one is a bit subtle - "the Katas" is the project name, while "the katas" is plural of "a kata", a single set of exercises. If we say lowercase katas here, we'd have to say "and tutorials", and the version update we're doing is really across everything, including configuration files.